app/cli/listapps.cpp

```diff
@@ -102,25 +102,36 @@ class LauncherPrivate
         // Occurs when a computer is updated
         case Event::ComputerUpdated:
             if (m_State == StateSeekApp) {
-                for (int i = 0; i < m_Computer->appList.length(); i++) {
-                    printApp(m_Computer->appList[i]);
-                }
+                m_PrintCSV ? printAppsCSV(m_Computer->appList) : printApps(m_Computer->appList);
 
                 QCoreApplication::exit(0);
             }
             break;
         }
     }
 
-    void printApp(NvApp app) const
+    void printApps(QVector<NvApp> apps) {
+        for (int i = 0; i < apps.length(); i++) {
+            fprintf(stdout, "%s\n", qPrintable(apps[i].name));
+        }
+    }
+
+    void printAppsCSV(QVector<NvApp> apps) {
+        fprintf(stdout, "Name, ID, HDR Support, App Collection Game, Hidden, Direct Launch, Boxart URL\n");
+        for (int i = 0; i < apps.length(); i++) {
+            printAppCSV(apps[i]);
+        }
+    }
+
+    void printAppCSV(NvApp app) const
     {
-        fprintf(stdout, "%s,%d,%s,%s,%s,%s,%s\n", qPrintable(app.name),
-                                               app.id,
-                                               app.hdrSupported ? "true" : "false",
-                                               app.isAppCollectorGame ? "true" : "false",
-                                               app.hidden ? "true" : "false",
-                                               app.directLaunch ? "true" : "false",
-                                               qPrintable(m_BoxArtManager->loadBoxArt(m_Computer, app).toDisplayString()));
+        fprintf(stdout, "\"%s\",%d,%s,%s,%s,%s,\"%s\"\n", qPrintable(app.name),
```

Comment:
One corner case remains if the App name contains a `"` character. I already had code in place to escape that. However, when trying to test, I was not able to have an App with `"` in its name. Therefore, I assume that `"` is not possible in an App name. Furthermore, because I was not able to test the code, I decided to leave it out